### PR TITLE
Finish very basic instruction for network mode installation

### PR DESCRIPTION
**Pull request by dm3ch**
/closes https://github.com/chainstack/graph-deployment/issues/15

### CODE REVIEW

File: helmfile/README.md [hunk 1]
```diff
@@ -5,22 +5,48 @@ Graph protocol indexer could be installed in two different modes:
...
+1. [kubectl](https://kubernetes.io/docs/tasks/tools/)
 2. [helmfile](https://github.com/roboll/helmfile)
 3. [helm-diff](https://github.com/databus23/helm-diff)
 
 Make sure to setup the kubectl config to point to the right kubernetes instance, see `terraform/<respective cloud provider>` for instructions.
 
 ## Network mode
 You can deploy graph node in network mode. In this case additional graph indexer components gets deployed, so that the node can join the graph network.
-
+This The Graph doc could be useful - https://thegraph.com/docs/indexing .
```
Comment (CaJIbHuK) [inline helmfile/README.md:16]:
```suggestion
More information about indexing could be found [here](https://thegraph.com/docs/indexing).
```

File: helmfile/README.md [hunk 2]
```diff
@@ -54,7 +80,7 @@ Should timeouts be experienced or if you noticed that some pods are experiencing
 it suggests that there are configuration errors.
 
 Some possible errors could be
-1. Blank fields in files found in `value`, if the field is not needed or optional. Remember to comment them out.
+1. Blank network providers found in `values/graphprotocol-node*` in `config.chains`, if the network wasn't configure don't remember to comment it out.
```
Comment (CaJIbHuK) [inline helmfile/README.md:83]: Tbh, didn't fully understand what you meant here. Maybe the following line describes what you want to say:

```suggestion
Don't forget to fill in the `config.chains` value in the `values/graphprotocol-node*` file.
```
+
maybe `Some possible errors could be` line can be removed.

File: helmfile/README.md [hunk 3]
```diff
@@ -5,22 +5,48 @@ Graph protocol indexer could be installed in two different modes:
...
 ### Installation
 Prepare values in `values` directory first. `<namespace>` can be any string of your choice.
 ```
 helmfile -f helmfile-network.yaml -n <namespace> apply
 ```
 
-**TODO: document full installation, setup and usage** - https://github.com/chainstack/graph-deployment/issues/15
+### Install graph-cli and indexer-cli
+It's needed to configure indexer-agent rules and check status
+https://thegraph.com/docs/indexing#from-npm-packages
```
Comment (CaJIbHuK) [inline helmfile/README.md:25]:
```suggestion
Please, install graph-cli and indexer-cli according to the following [guide](https://thegraph.com/docs/indexing#from-npm-packages).
```

File: helmfile/README.md [hunk 4]
```diff
@@ -5,22 +5,48 @@ Graph protocol indexer could be installed in two different modes:
...
+
+### Check indexer status
+```
+kubectl -n <namespace> port-forward svc/graphprotocol-indexer-agent 18000:8000
+graph indexer status
+```
+
+### Interacting with indexer-agent
+First you need to run port-forward command:
```
Comment (CaJIbHuK) [inline helmfile/README.md:34]:
```suggestion
First, you need to run the port-forward command:
```

File: helmfile/README.md [hunk 5]
```diff
@@ -5,22 +5,48 @@ Graph protocol indexer could be installed in two different modes:
...
+```
+kubectl -n <namespace> port-forward svc/graphprotocol-indexer-agent 18000:8000
+```
+
+Connect cli:
```
Comment (CaJIbHuK) [inline helmfile/README.md:39]:
```suggestion
Then, connect with graph-cli and check status:
```

File: helmfile/README.md [hunk 6]
```diff
@@ -5,22 +5,48 @@ Graph protocol indexer could be installed in two different modes:
...
+```
+graph indexer connect http://localhost:18000
+```
+
+Request indexer status:
+```
+graph indexer status
```
Comment (CaJIbHuK) [inline helmfile/README.md:46]:
```suggestion
graph indexer connect http://localhost:18000
graph indexer status
```

File: helmfile/README.md [hunk 7]
```diff
@@ -5,22 +5,48 @@ Graph protocol indexer could be installed in two different modes:
...
+```
+
+Other useful commands for subgraphs and cost model configuration could be found there: https://thegraph.com/docs/indexing#usage-1
```
Comment (CaJIbHuK) [inline helmfile/README.md:49]:
```suggestion
Other useful commands for subgraphs and cost model configuration could be found [here](https://thegraph.com/docs/indexing#usage-1).
```

**Review by CaJIbHuK — CHANGES_REQUESTED** [review]

**Review by CaJIbHuK — APPROVED** [review]

**Comment by CaJIbHuK** [comment]
@akegaviar please look through

**Review by akegaviar — APPROVED** [review]

**Pull request merged**